scripts/.bashrc_patch

```diff
@@ -1,3 +1,9 @@
-if [[ -z "${NO_SETUP}" ]]; then
-  source entry.sh
+export ENTITIES="/home/node/ardrive-cli/.bats/entities.sh"
+
+if [[ -z "${NO_SETUP}" ]] && [[ ! -f "/home/node/ardrive-cli/package.json" ]]; then
+    source /home/node/entry.sh
+fi
+
+if [[ -f "${ENTITIES}" ]] && [[ -f "${WALLET}" ]] && [[ $PUB_DRIVE_ID == '00000000-0000-0000-0000-000000000000' ]]; then
+    source /home/node/ardrive-cli/.bats/entities.sh
```

Comment:
Use the constant declared at the top here

Reply:
Fixed